tox.ini

```diff
@@ -20,8 +20,8 @@ allowlist_externals=/usr/bin/make
 install_command=python -m pip install {opts} {packages}
 usedevelop=True
 commands=
-    core: pytest {posargs:tests/core -k "not async"}
-    core_async: pytest {posargs:tests/core -k async}
+    core: pytest {posargs:tests/core -m "not asyncio" -k "not async"}
+    core_async: pytest {posargs:tests/core -k "asyncio or async"}
```

Comment:
Should `asyncio` be part of the mark expression flag? I think this will find any test names containing `asyncio` or `async` (which in turn finds anything that would include `asyncio` I believe).